diagrams/balance-without-credit-card.svg

```diff
@@ -0,0 +1,3 @@
BAgAABAgQIECBAgAABAgQIECBAgAABArUTEDir3ZTpMAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBDojIHDWGXdPJUCAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAQO0EBM5qN2U6TIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAgc4ICJx1xt1TCRAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgUDsBgbPaTZkOEyBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAoDMCAmedcfdUAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQI1E5A4Kx2U6bDBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQ6IyAwFln3D2VAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECtRMQOKvdlOkwAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIEOiMgcNYZd08lQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIBA7QQEzmo3ZTpMgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgACBzggInHXG3VMJECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBAgAABAgQIECBQOwGBs9pNmQ4TIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECgMwICZ51x91QCBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAgQIECAAAECBAjUTkDgrHZTpsMECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBAgQIAAAQIECBDojMD/D1Hr4j5cPUDEAAAAAElFTkSuQmCC" preserveAspectRatio="none"/><path d="M 558.05 320 L 558.05 371.9" fill="none" stroke="#ff0000" stroke-width="3" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 558.05 378.65 L 553.55 369.65 L 558.05 371.9 L 562.55 369.65 Z" fill="#ff0000" stroke="#ff0000" stroke-width="3" stroke-miterlimit="10" pointer-events="all"/><rect x="272.09" y="187.65" width="571.91" height="132.35" fill="none" stroke="#ff0000" stroke-width="3" pointer-events="all"/><rect x="272.09" y="382" width="571.91" height="45" fill="none" stroke="#ff0000" stroke-width="3" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 570px; height: 1px; padding-top: 405px; margin-left: 273px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font face="Verdana">This is where you add your credit card with FuelStop. Your card will be charged immediately by the default "Top Up Amount", which is $10.</font></div></div></div></foreignObject><text x="558" y="408" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">This is where you add your credit card with FuelStop. Your card will be charged immediately by...</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://desk.draw.io/support/solutions/articles/16000042487" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
This is where you add your credit card to your project. Your card will be charged immediately by the default "Top Up Amount", which is $10.